src/components/alert/alert.svelte

```diff
@@ -19,6 +19,7 @@
   export let type: AlertType = 'error'
   export let mode: AlertMode = 'simple'
   export let isToast = false
+  export let hasActions = $$slots.actions
```

Comment:
Can't say I'm proud of this, but it's the only way I can see to avoid the empty space when no actions are present for the `<AlertCenter>` given that we can't wrap the `slot="actions"` in a conditional: https://github.com/brave/leo/blob/d296d0335219c8daa942829a28033fdcbd0552b7/src/components/alert/alertCenter.svelte#L101

Reply:
I think, instead of doing this you could wrap the `<div slot="actions">` in `AlertCenter` in an `#if actions.length` block

Reply:
Wait, why can't we wrap that in a conditional?

Reply:
It's a big bummer, but Svelte doesn't allow it. Allegedly, `snippets` will solve this problem.

https://github.com/sveltejs/svelte/issues/5604#issuecomment-1808036921

![Screenshot 2024-01-16 at 10 50 22 AM](https://github.com/brave/leo/assets/1199820/83a26bd1-6e4d-43b0-8398-ef1fd7a87341)

Reply:
Wow.... sometimes I really wonder if we should be using Svelte :laughing: